1-Authentication/5-sign-in-express/README.md

```diff
@@ -37,163 +39,97 @@ extensions:
 
 ## Contents
 
 | File/folder           | Description                                |
 |-----------------------|--------------------------------------------|
 | `App/app.js`          | Application entry point.                   |
-| `App/authConfig.js`   | Contains authentication parameters.        |
-| `App/auth/AuthProvider.js`  | Main authentication logic resides here.    |
+| `App/authConfig.js`   | Contains authentication parameters such as your tenant sub-domain, Application (Client) ID, app client secret and redirect URI.        |
+| `App/auth/AuthProvider.js`  | The main authentication logic resides here.    |
+|    `/App/views/`    |    This folder contains app views. This Node/Express sample app's views uses Handlebars. |
+|    `/App/routes/`    |    This folder contains app's routes. |
 
 ## Prerequisites
 
-* [Node.js](https://nodejs.org/en/download/) must be installed to run this sample.
-* [Visual Studio Code](https://code.visualstudio.com/download) is recommended for running and editing this sample.
-* [VS Code Azure Tools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-node-azure-pack) extension is recommended for interacting with Azure through VS Code Interface.
-* An **Azure AD for Customers** tenant. For more information, see: [How to get an Azure AD for Customers tenant](https://github.com/microsoft/entra-previews/blob/PP2/docs/1-Create-a-CIAM-tenant.md)
-* A user account with permissions to create applications in your **Azure AD for Customers** tenant.
-
-## Setup the sample
-
-### Step 1: Clone or download this repository
-
-From your shell or command line:
-
-```console
-git clone https://github.com/Azure-Samples/ms-identity-ciam-javascript-tutorial.git
-```
-
-or download and extract the repository *.zip* file.
-
-> :warning: To avoid path length limitations on Windows, we recommend cloning into a directory near the root of your drive.
+* You must install [Node.js](https://nodejs.org/en/download/) in your computer to run this sample.
+* We recommend [Visual Studio Code](https://code.visualstudio.com/download) for running and editing this sample.
+* Microsoft Entra ID for customers tenant. If you don't already have one, [sign up for a free trial](https://aka.ms/ciam-free-trial).
+* If you'd like to use Azure services, such as hosting your app in Azure App Service, [VS Code Azure Tools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-node-azure-pack) extension is recommended for interacting with Azure through VS Code Interface.
 
-### Step 2: Install project dependencies
-
-```console
-    cd 1-Authentication\5-sign-in-express\App
-    npm install
-```
+## Register the web application in your tenant
 
-### Step 3: Register the sample application(s) in your tenant
+You can register an app in your tenant automatically by using Microsoft Graph PowerShell or Manually register your apps in Microsoft Entra Admin center.
```

Comment:
is it possible to put the instructions in bold for powershell and for entra admin. i was trying the steps and got confused which are entra admin steps and which are powershell steps.

Reply:
Not possible, but I have changed formatting 

<img width="749" alt="image" src="https://github.com/Azure-Samples/ms-identity-ciam-javascript-tutorial/assets/88311008/f6b5e7b2-d790-46e5-a679-c131d6d8c979">